tests/rptest/services/redpanda.py

```diff
@@ -5350,7 +5350,8 @@ def install_license(self):
         def license_observable():
             for node in self.started_nodes():
                 license = self._admin.get_license(node)
-                if license is None or license['loaded'] is not True:
+                if license is None or not license['loaded'] or \
+                        license['license']['type'] == 'free_trial':
```

Comment:
The license type should be reported as `trial`.  Another check would be the organization or if the checksum returned is empty.  cc @pgellert 

Maybe this will be fixed when you rebase off of Gellert's changes.

Reply:
yeah, skipped this commit during rebase.